.github/workflows/test-upstream-wpt-changes.yml

```diff
@@ -6,6 +6,8 @@ on:
 
 jobs:
   test:
+    # Run job only on servo/servo
+    if: github.repository == 'servo/servo'
```

Comment:
I would say that it does make sense to run this particular action on forks, because it would allow someone to verify that the changes they made to this script actually pass CI.